GoMeet/src/main/java/com/google/sps/data/Dao.java

```diff
@@ -0,0 +1,20 @@
+package main.java.com.google.sps.data;
+
+import com.google.appengine.api.datastore.EntityNotFoundException;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface Dao<T> {
```

Comment:
Is this going to be a general class that we will all use? If not, it might be better to name it something more specific to the location feature

Reply:
I think it would be best to keep the name non-specific (so Dao instead of LocationDao) because it is a generic interface. For your question, we don't all have to use it. It is up to you if you would like to use it and what you think is best for your feature (I found that this was useful for me as a I had multiple servlets accessing the same data).

Reply:
Ok that makes sense, agreed I think this was a good style choice for your implementation :)